src/modules/get.js

```diff
@@ -0,0 +1,14 @@
+export const fetchData = async () => fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/iLDJ7NaS4vrrLkwwbZmu/scores/')
+  .then((response) => response.json());
+export const postScore = async (user, score) => {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/iLDJ7NaS4vrrLkwwbZmu/scores/', {
+    method: 'POST',
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+};
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update the content of the `Setup` subsection by adding the appropriate content on how to run your app in the server and but before hoe run the build for production. i.e.
```
...
## Run build for production
npm run build

## Usage
npm start
...
```